### ISSUE TITLE
kfp.Client fails to work in master version

### ISSUE THREAD

**Issue by Ark-kun** [kubeflow/pipelines#4847]
I've installed kfp from the repository. The presubmit tests should ensure that the master version works.
However I've encountered the following error:
```
$ python -c "import kfp; kfp.Client()"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "a:\_all\my.work\2018.google\pipelines_worktree2\sdk\python\kfp\_client.py", line 147, in __init__
    self._healthz_api = kfp_server_api.api.healthz_service_api.HealthzServiceApi(api_client)
AttributeError: module 'kfp_server_api.api' has no attribute 'healthz_service_api'
```
I expect that the kfp package installed from master works and can pass tests.

This might be related to https://github.com/kubeflow/pipelines/pull/4638
/cc @DavidSpek

**Comment by DavidSpek** [comment]
As I mentioned elsewhere, I suspect the issue is a mismatch between the kfp client and kfp-api-server. It would be my expectation that installing from master is known to work with the other components from the master branch. When looking at the releases page, it seems kfp and kfp-api-server are always installed together, so I think this should also be done when installing from the master branch. I am currently using kfp and kfp-api-server Version 1.1.1-beta.1, while still running the older KFP 1.0.4 deployment from kubeflow 1.2 in the cluster and I am not getting any errors from jupyter notebooks I am running.

**Comment by numerology** [comment]
I believe it's the mismatched version between two packages as well. Can you share the result of `pip freeze`? @Ark-kun

**Comment by Bobgy** [comment]
Agree, I think when installing `kfp` from master, you should also install `kfp-server-api` from master.

**Comment by Bobgy** [comment]
That reminds me that, we should update `kfp` package's version dependency on `kfp-server-api`.

**Comment by Bobgy** [comment]
Problem fixed in #4855
